common/source/docs/common-heewingf405.rst

```diff
@@ -0,0 +1,166 @@
+The RCIN pin, is passed by an inverter to RX2 (UART2 RX), which by default is mapped to a timer input instead of the UART, and can be used for all ArduPilot supported receiver protocols, except CRSF/ELRS and SRXL2 which require a true UART connection. However, FPort, when connected in this manner, can provide RC without telemetry. 
+
+To allow CRSF and embedded telemetry available in Fport, CRSF, and SRXL2 receivers, the RX2 pin can also be configured to be used as true UART2 RX pin for use with bi-directional systems by setting the :ref:`BRD_ALT_CONFIG<BRD_ALT_CONFIG>` to “1” so it becomes the SERIAL2 port's RX input pin.
+
+With this option, :ref:`SERIAL6_PROTOCOL<SERIAL2_PROTOCOL>` must be set to "23", and:
+
+- PPM/SBUS/DSM/SRXL is not supported.
+
+- FPort requires connection to TX2 and RX2 via a bi-directional inverter. See :ref:`common-FPort-receivers`.
+
+- CRSF also requires a TX2 connection, in addition to RX2, and automatically provides telemetry.
+
+- SRXL2 requires a connection to TX2 and automatically provides telemetry.  Set :ref:`SERIAL6_OPTIONS<SERIAL2_OPTIONS>` to "4".
+
+.. note:: UART2 is configured by default for serial receivers. You can also have more than one receiver in the system at a time (usually used for long range hand-offs to a remote TX). See :ref:`common-multiple-rx` for details.
+
+Any UART can be used for RC system connections in ArduPilot also, and is compatible with all protocols except PPM (SBUS requires external inversion on other UARTs). See :ref:`common-rc-systems` for details.
+
+OSD Support
+===========
+
+The HEEWINGF405 supports using its internal OSD using OSD_TYPE 1 (MAX7456 driver). External OSD support such as DJI or DisplayPort is supported using any free UART. See :ref:`common-msp-osd-overview-4.2` for more info.
+
+Battery Monitor Configuration
+=============================
+These settings are set as defaults when the firmware is loaded for use with its accompanying PMU. However, if they are ever lost, you can manually set the parameters:
+
+Enable Battery monitor.
+
+:ref:`BATT_MONITOR<BATT_MONITOR>` =4
+
+Then reboot.
+
+:ref:`BATT_VOLT_PIN<BATT_VOLT_PIN>` 10
+
+:ref:`BATT_CURR_PIN<BATT_CURR_PIN>` 11
+
+:ref:`BATT_VOLT_MULT<BATT_VOLT_MULT>` 7.71
+
+:ref:`BATT_AMP_PERVLT<BATT_AMP_PERVLT>` 26.7
+
+Connecting a GPS/Compass module
+===============================
+
+This board does not include a GPS or compass so an :ref:`external GPS/compass <common-positioning-landing-page>` should be connected in order for autonomous modes to function.
+
+Firmware
+========
+This board does not come with ArduPilot firmware pre-installed (it may at some future point). Because it has not boot pin exposed, DFU loading cannot be used to load standard ArduPilot firmware. Instead a `"transitional" firmware <https://firmware.ardupilot.org/Tools/TransitionalFirmware/HeeWingF405/>`__  can be used and loaded like normal ArduPilot firmware (this is rev 4.3.7 Plane firmware) as a custom firmware file in Mission Planner.
```

Comment:
Actually this is not true - it *does* come with AP pre-installed, it's just the board ID they have used is for the MatekF405 so upgrades are not possible. The transitional firmware allows upgrades

Reply:
Ill say it comes with an incorrect bootloader which prevents ArduPilot firmware changes.